lint/ilib-lint-config.json

```diff
@@ -1,37 +1,72 @@
 {
-    "name": "lint-test",
-    "plugins": [
+  "name": "lint-test",
+  "plugins": [
+    "python",
+    "python-gnu",
+    "react"
+  ],
+  "locales": [
+    "en-US",
+    "de-DE"
+  ],
+  "filetypes": {
+    "jsx": {
+      "parsers": [
+        "JSXParser"
+      ],
+      "ruleset": [
+        "generic",
+        "react",
+        "source"
+      ]
+    },
+    "python": {
+      // "parsers": [ "PythonParser" ],
+      "ruleset": [
+        "generic",
         "python",
         "python-gnu",
-        "react"
-    ],
-    "locales": [
-        "en-US",
-        "de-DE"
-    ],
-    "filetypes": {
-        "jsx": {
-            "parsers": [ "JSXParser" ],
-            "ruleset": [ "generic", "react", "source" ]
-        },
-        "python" : {
-            // "parsers": [ "PythonParser" ],
-            "ruleset": [ "generic", "python", "python-gnu", "source" ]
-        },
-        "xliff": {
-            "ruleset": [ "generic", "python", "python-gnu" ]
-        }
-    },
-    "paths": {
-        "xliffs/*.xliff": "xliff",
-        "src/**/*.py": "python",
-        "src/**/*.jsx": "jsx"
+        "source"
+      ]
     },
-    "excludes": [
-        "**/.git",
-        "**/node_modules",
-        "**/.svn",
-        "package.json",
-        "package-lock.json"
-    ]
+    "xliff": {
+      "ruleset": [
+        "generic",
+        "python",
+        "python-gnu",
+        "tap"
+      ]
+    }
+  },
+  "paths": {
+    "xliffs/*.xliff": "xliff",
+    "src/**/*.py": "python",
+    "src/**/*.jsx": "jsx"
+  },
+  "excludes": [
+    "**/.git",
+    "**/node_modules",
+    "**/.svn",
+    "package.json",
+    "package-lock.json"
+  ],
+  "rules": [
+    {
+      "type": "resource-matcher",
+      "name": "resource-camel-case",
```

Comment:
The rule should not be defined here. If it changes in the linter itself, then this one will not change and we will not catch problems.